### PR TITLE
Feature/indirajhenny

### PR DESCRIPTION
**Pull request by indirajhenny**
### Summary

This is the updated version on the Workshops Schedule component. I made updates to the design and switched from Firebase firestore access code to realtime access code. The scss for both WkspShedule and WkspCard also successfully compiled with sass. I also scaffolded the live page with the updated workshops component. 
### Screenshots/GIFs
Screen Shot 2020-01-05 at 2 35 42 AM

### Responsiveness

If the page gets minimized, the text inside the Workshop Cards starts to overlap into other cards and it looks messy. Should I look into a way to prevent that from happening? This is when the page is much more minimized: 
Screen Shot 2020-01-05 at 11 53 38 AM

### Questions or Blockers?

I am having trouble converting the Timestamp data type for the workshop Start Time to a string or Date type that can properly show on the design component. So far every time I try to display "startTime" the component cannot compile. 

@brianbaso

### CODE REVIEW

File: src/components/FeedbackModal.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                                At Knight Hacks, we find it extremely important to collect as much
+                                feedback as we possibly can from those in attendance at our events.
+                                For this reason, we look forward to hearing how we're doing from you!
+                            </p>
+                            <h6>Current Rating</h6>
+                            <p style={{ marginBottom: '0px' }}>
+                                Please indicate below what your current rating of the event would be?
+                            </p>
+                            <ToggleButtonGroup type="radio" name="rating" onClick={(e) => this.onRatingChange(e)} style={{ width: '100%', margin: '10px 0' }}>
+                                <ToggleButton className="ToggleButton tall" value={1}>1</ToggleButton>
```
Comment (nalshihabi) [inline src/components/FeedbackModal.js]: You can probably make this a simpler thing by making a helper method called `renderToggleButtons` or something similar and use a function such as `map`. [Here is a good example](https://stackoverflow.com/questions/22876978/loop-inside-react-jsx). I don’t like the selected answer but instead the second answer. It is more elegant and more of a practice I have seen in React in the situation you have here. You can make like I said that contains the `map` call and that will condense this code and make it much prettier.

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,40 @@
...
 
 function App() {
   return (
     <div className="App">
       <Header />
-      {/* <LiveUpdates /> */}
+      <div className="row no-gutters mainHolder">
+        <div className="col-12 col-xl-6">
+          <div style={{ height: '550px', maxWidth: '100%', borderRadius: '5px', margin:'10px', backgroundColor: 'white'}}>
+            <WorkshopSchedule />
```
Comment (nalshihabi) [inline src/App.js]: I am not super familiar with the code here so forgive me for asking this, but why is `WorkshopsSchedule` element called multiple times here?

**Comment by nalshihabi** [comment]
Also, since the changes here seem to replace the changes made in the other PR (unless I am mistaken), you can go ahead and close the other PR, since these changes will contain the other’s.

**Comment by brianbaso** [comment]
Whats going on with this PR @indirajhenny? Looks like there are merge conflicts, let me know if I can help with those.

**Pull request merged**